webhook/certificates/certificates_test.go

```diff
@@ -38,6 +39,21 @@ import (
 	. "knative.dev/pkg/webhook/testing"
 )
 
+var blockingFinished = []func(*testing.T, *TableRow){
+	func(t *testing.T, _ *TableRow) {
+		if s := webhook.SecretsReconciled; s != nil {
+			// We expect webhook.SecretsReconciled to be set to `nil` after
+			// reconciliation, so this should be the main case.
+			select {
+			case _ = <- webhook.SecretsReconciled:
```

Comment:
```suggestion
			case <- s:
```
I guess the channel might become nil, but it seems they're in the same func, so 🤷‍♂ .
But definitely don't need `_ := `.